net/p910nd/files/p910nd.hotplug

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+user=
+
+. /lib/functions.sh
+
+config_load p910nd
+config_get user p910nd user lp
+
+case "$ACTION" in
+        add)
+		[ "$DEVNAME" = "usb/lp0" ] && {
```

Comment:
Why only `lp0`?

Reply:
Tested and pushed update that fixes this!  Thanks for catching that.